netci.groovy

```diff
@@ -10,7 +10,17 @@ def project = GithubProject
 def branch = GithubBranchName
 def isPR = true
 
-def platformList = ['Linux:x64:Release', 'Linux:arm:Release', 'Linux:arm64:Release', 'OSX:x64:Release', 'Windows_NT:x64:Release', 'Windows_NT:x86:Debug', 'Windows_NT:arm:Debug', 'Tizen:armel:Release']
+def platformList = [
+  'Linux:x64:Release',
+  'Linux:arm:Release',
+  'Linux:arm64:Release',
+  'OSX:x64:Release',
+  'Windows_NT:x64:Release',
+  'Windows_NT:x86:Debug',
```

Comment:
I'm not familiar with this repo, but why are some Debug and some Release?

Reply:
I was surprised when I first saw this too.  My hypothesis is that the code in this repo is not very architecture specific.  It is pretty generic C++ code and therefore does not need a lot of per architecture testing.  So it is a bit of a random selection...